### PR TITLE
included chromedrivers, included max_liked_post example, added new pa…

### LINKED ISSUE #18: Examples
Provide simple and/or complex examples showcasing the different ways you can use instascrape. 

Jupyter Notebooks explaining your examples in detail are encouraged but not required!

### PR DESCRIPTION
**Pull request by Paola351**
…rameter to dynamic_load; added requirements

Thanks for contributing to instascrape! Please review and fill out the checklist below before submitting your request. 

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [] Have you written new tests for your changes, as applicable?
* [] Have you successfully ran tests with your changes locally?

### Additional Notes 
Feel free to include any additional information regarding your PR below: 

Fixes https://github.com/chris-greening/instascrape/issues/18

### CODE REVIEW

**Comment by chris-greening** [comment]
Hey @Paola351 !!! Awesome work, this is a great example 😄. I have a couple comments/suggestions to make before merging to `master` that I'll get to in a little bit (nothing is wrong at all, just a couple minor changes).

**Comment by Paola351** [comment]
> Hey @Paola351 !!! Awesome work, this is a great example 😄. I have a couple comments/suggestions to make before merging to `master` that I'll get to in a little bit (nothing is wrong at all, just a couple minor changes).

Thx :D Let me know if i can do something else :D

File: requirements.txt
Comment (chris-greening) [inline requirements.txt]: Can you remove the `selenium` dependency? I'm going to make a README.md for [examples/](https://github.com/chris-greening/instascrape/tree/master/examples) and I'll mention in there that it's required for some of the examples but not for general usage with instascrape

File: examples/profile_scrape/dynamic_profile.py
Comment (chris-greening) [inline examples/profile_scrape/dynamic_profile.py]: lol definitely a good idea generalizing this, I honestly have no idea what I was thinking leaving my chromedriver path for the user to have to change 🤣  

Instead of having the chromedrivers in the repo though, do you think you could yank those files out and leave it up to the user to pass their own browser object or `'/path/to/driver.exe'`? I figure for maintainability (and security) we can leave it up to them to either
- pass the filepath to their chromedriver with something like 
```python
def dynamic_load(self, executable_path, max_posts=sys.maxsize)
```
where `executable_path` is the path to their chromedriver  
or alternatively (and perhaps preferably)
- pass their own browser object with something like 
```python
def dynamic_load(self, browser, max_posts=sys.maxsize)
```
where `browser` is a Selenium webdriver of their choice

Either one works for me so that's totally up to you with what you decide (or if you think of a better alternative)! 
If this is confusing or you don't feel like changing it tho then no worries lol I don't mind making the change

**Review by chris-greening — CHANGES_REQUESTED** [review]

File: examples/profile_scrape/dynamic_profile.py
Comment (Paola351) [reply examples/profile_scrape/dynamic_profile.py]: I don't know if there could be some security issue with that but, as you asked, I'm going to implement the suggestions :D I thought it could be helpful have the drivers directly in the project for those who run the test to understand the potentiality of the library :) I prefer passing the browser preference directly :D

File: requirements.txt
Comment (Paola351) [reply requirements.txt]: Sure!

**Comment by Paola351** [comment]
@chris-greening review this when you can please :D

**Comment by chris-greening** [comment]
Awesome work, thank you so much for the contribution!!!! 😄

**Pull request merged**
